zap/src/main/java/org/zaproxy/zap/control/AddOn.java

```diff
@@ -443,13 +442,14 @@ public static Optional<AddOn> createAddOn(Path file) {
             return Optional.of(new AddOn(file));
         } catch (AddOn.InvalidAddOnException e) {
             String logMessage = "Invalid add-on: " + file.toString() + ".";
-            if (logger.isDebugEnabled() || Constant.isDevMode()) {
+            if (Constant.isDevMode()) {
                 logger.warn(logMessage, e);
             } else {
-                logger.warn(logMessage + " " + e.getMessage());
+                logger.warn(logMessage);
+                logger.warn(e.getMessage());
```

Comment:
Maybe this could be:
```suggestion
                logger.warn(logMessage, e);
```

Reply:
That changes behaviour.

Reply:
So leave my change?

Reply:
Actually I've changed this to `logger.warn("{} {}", logMessage, e.getMessage());` which I believe is closest to the original.